pyglet/lib.py

```diff
@@ -250,6 +250,16 @@ def find_library(self, path):
     @staticmethod
     def load_framework(name):
         path = ctypes.util.find_library(name)
+
+        # Hack for compatibility with macOS > 11.0
+        if path is None and sys.platform == 'darwin':
+            frameworks = {
+                'AGL': '/System/Library/Frameworks/AGL.framework/AGL',
+                'OpenAL': '/System/Library/Frameworks/OpenAL.framework/OpenAL',
+                'OpenGL': '/System/Library/Frameworks/OpenGL.framework/OpenGL'
+            }
+            path = frameworks.get(name)
+
         if path:
```

Comment:
I suppose the `if path` check is never reached now, since we are explicitly setting it above. If we're planning on removing that hack later, then maybe we can just leave this alone though.

Reply:
Yeah I left it for 2 reasons, the first being that hopefully we will be able to remove this someday. The other being that if for some reason we decided to load a framework other than the 3 in the dictionary above for some reason, I wanted it to behave the same way it would if the dictionary wasn't there.